WebDriverAgentLib/Commands/FBFindElementCommands.m

```diff
@@ -47,6 +47,9 @@ + (NSArray *)routes
     [[FBRoute POST:@"/element/:uuid/element"] respondWithTarget:self action:@selector(handleFindSubElement:)],
     [[FBRoute POST:@"/element/:uuid/elements"] respondWithTarget:self action:@selector(handleFindSubElements:)],
     [[FBRoute GET:@"/wda/element/:uuid/getVisibleCells"] respondWithTarget:self action:@selector(handleFindVisibleCells:)],
+#if TARGET_OS_TV
+    [[FBRoute GET:@"/wda/element/focused"] respondWithTarget:self action:@selector(handleFindFocusedElement:)],
```

Comment:
this looks like a duplicate of the endpoint above

Reply:
Here gets a focused element in the display. So, it is similar to `/element/active`.